roles/prometheusexporter/templates/grafanadashboard-es.yml.j2

```diff
@@ -2012,54 +2012,775 @@ spec:
             "align": false,
             "alignLevel": null
           }
-        }
-      ],
-      "refresh": "30s",
-      "schemaVersion": 18,
-      "style": "dark",
-      "tags": [
-        "prometheus-exporter",
-        "es"
-      ],
-      "templating": {
-        "list": [
-          {
-            "hide": 0,
-            "includeAll": false,
-            "label": null,
-            "multi": false,
-            "name": "datasource",
-            "options": [],
-            "query": "prometheus",
-            "refresh": 1,
-            "regex": "",
-            "skipUrlSync": false,
-            "type": "datasource"
+        },
+        {
+          "collapsed": false,
+          "datasource": null,
+          "gridPos": {
+            "h": 1,
+            "w": 24,
+            "x": 0,
+            "y": 61
           },
-          {
-            "allValue": null,
-            "current": {
-              "text": "{{ meta.namespace }}",
-              "value": "{{ meta.namespace }}"
+          "id": 103,
+          "panels": [],
+          "title": "System",
+          "type": "row"
+        },
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": null,
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 12,
+            "x": 0,
+            "y": 62
+          },
+          "hiddenSeries": false,
+          "id": 101,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "dataLinks": []
+          },
+          "percentage": false,
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "(elasticsearch_filesystem_data_size_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"} - elasticsearch_filesystem_data_free_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"}) / elasticsearch_filesystem_data_size_bytes{cluster=~'.*:pro-base-es-logs', es_master_node=\"true\"} * 100",
```

Comment:
All queries should include minimum namespace and prometheus-exporter label (to differenciate among possible elasticsearch exporters deployed on same namespace or others, something like the following (extracted from an example expression above):
```              
"expr": "sum(elasticsearch_thread_pool_active_count{namespace=\"$namespace\", prometheus_exporter=\"$prometheus_exporter\", type!=\"management\"}) by (type)",
"format": "time_series",
```

And should'nt include something harcoded like `cluster=~'.*:pro-base-es-logs'`